src/Data/Product/Function/Dependent/Propositional.agda

```diff
@@ -316,3 +316,8 @@ cong {B = B} {k = reverseInjection}   I↔J A↢B = Σ-↣ (↔-sym I↔J) (swap
 cong {B = B} {k = leftInverse}        I↔J A↩B = ↩⇒↪ (Σ-↩ (↔⇒↩ (↔-sym I↔J)) (↪⇒↩ (swap-coercions {k = leftInverse} B I↔J A↩B)))
 cong {k = surjection}                 I↔J A↠B = Σ-↠ (↔⇒↠ I↔J) A↠B
 cong {k = bijection}                  I↔J A↔B = Σ-↔ I↔J A↔B
+
+cong′ : ∀ {k} → {A : Set a} {B₁ : A → Set b} {B₂ : A → Set c} →
```

Comment:
Can we use `I` for the index type instead of `A`, and `A` and `B` instead of `B₁`/`B₂`. Then we can use the variables and not have to add them to the type directly.